# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/rustc/hotfix-crate-id.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

error: field is never read: `id`
   --> src/bootstrap/lib.rs:280:5
    |
280 |     id: String,
    |     ^^^^^^^^^^
    |
    = note: `-D dead-code` implied by `-D warnings`

error: could not compile `bootstrap` due to previous error

--- rustc-1.57.0-src/src/bootstrap/lib.rs.vanilla	2022-01-11 18:38:46.864951893 +0100
+++ rustc-1.57.0-src/src/bootstrap/lib.rs	2022-01-11 18:39:17.971950653 +0100
@@ -274,6 +274,7 @@
 }
 
 #[derive(Debug)]
+#[allow(dead_code)]
 struct Crate {
     name: Interned<String>,
     deps: HashSet<Interned<String>>,
